### PR TITLE
Create andbeyondBidAdapter.js

### PR DESCRIPTION
**Pull request by andbeyondmedia**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [x] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

- test parameters for validating bids
```
{
  bidder: '<bidder name>',
  params: {
    // ...
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
- [ ] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

**Comment by andbeyondmedia** [comment]
Adding new adapter

**Comment by andbeyondmedia** [comment]
can you please merge this

**Comment by jsnellbaker** [comment]
Hi @andbeyondmedia, thanks for submitting this new adapter.  Can you please update this PR by including the following two types of files?
- an .md file that contains a copy of your adapter's test param information
- a `adapterName_spec.js` unit test file that covers your adapter

The following page provides some additional information on these two pieces:
http://prebid.org/dev-docs/bidder-adaptor.html#adding-unit-tests

For the .md examples, you can refer to the .md files for other adapters to get an idea.

**Comment by andbeyondmedia** [comment]
added https://github.com/prebid/Prebid.js/pull/2457 added andbeyondBidAdapter_spec.js
https://github.com/prebid/prebid.github.io/pull/727 added andbeyond.md

please merge

**Comment by jsnellbaker** [comment]
@andbeyondmedia Are you able to merge/incorporate these files into 1 PR instead of separate PRs?  Having all the needed files in one PR makes it easier to maintain the changes and test them together.

Further, the .md file is needed in this repo in addition the one you created in the docs repo.  Can you please put together a .md file with the test data I can use for testing?  Below are some samples if you want to refer to something:
- https://github.com/prebid/Prebid.js/blob/master/modules/appnexusBidAdapter.md
- https://github.com/prebid/Prebid.js/blob/master/modules/rubiconBidAdapter.md

File: modules/andbeyondBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+        prBid.height = imp.video.h;
+      }
+      return prBid;
+    });
+  },
+  getUserSyncs: function(syncOptions, serverResponses) {
+    if (!syncOptions.iframeEnabled || !serverResponses || serverResponses.length === 0) {
+      return [];
+    }
+    return serverResponses.filter(rsp => rsp.body && rsp.body.ext && rsp.body.ext.adk_usersync)
```
Comment (jsnellbaker) [inline modules/andbeyondBidAdapter.js]: Just to confirm - is `adk_usersync` the correct field for this adapter?

**Pull request closed without merging**